### PR TITLE
CLI support for exporting all rounds of a game in JSON

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

**Comment by dionyziz** [comment]
Depends on #262

**Comment by dionyziz** [comment]
@pkakelas please review

File: controllers/game.php [hunk 1]
```diff
@@ -61,13 +61,20 @@ public function view( $gameid, $roundid = false ) {
...
             }
-            if ( isset( $this->acceptTypes[ 'application/json' ] ) ) {
-                require_once 'models/grader/serializer.php';
-                $mapJson = GraderSerializer::serializeCreatureList( $round->creatures );
-                require 'views/game/view.json.php';
-            }
-            else {
-                require 'views/game/view.php';
+            switch ( $this->outputFormat ) {
+                case 'json':
```
Comment (pkakelas) [inline controllers/game.php:65]: Are you sure that you want this case to be empty? Then why you set it?

Comment (vitsalis) [inline controllers/game.php:65]: it's not empty. I don't have a `break` here so if `outputFormat` is json, it executes the code from line 67.

**Pull request merged**